PhysiCell/config/PhysiCell_settings.xml

```diff
@@ -101,7 +101,7 @@
                     <model code="101" name="necrosis">
                         <death_rate units="1/min">0.0</death_rate>
                         <phase_transition_rates units="1/min">
-                            <rate start_index="0" end_index="1" fixed_duration="false">9e9</rate>
+                            <rate start_index="0" end_index="1" fixed_duration="false">9000000000.0</rate>
```

Comment:
This was changed automatically when I launched model builder, so I will keep it like this.

Reply:
yes, i know, i think the studio does this. no worries!